.github/workflows/ci.yml

```
@@ -631,6 +631,15 @@ jobs:
             docs-inventory-
       - name: "Build docs"
         run: breeze build-docs ${{ needs.build-info.outputs.docs-filter-list-as-string }}
+      - name: "Clone airflow-site"
+        run: >
+          git clone https://github.com/apache/airflow-site.git ${GITHUB_WORKSPACE}/airflow-site &&
+          echo "AIRFLOW_SITE_DIRECTORY=${GITHUB_WORKSPACE}/airflow-site" >> "$GITHUB_ENV"
+      - name: "Publish docs"
+        run: >
+          breeze release-management publish-docs
+          --override-versioned
+          --package-filter ${{ needs.build-info.outputs.docs-filter-list-as-string }}
```

```suggestion
          ${{ needs.build-info.outputs.docs-filter-list-as-string }}
```

The `--package-filter` is already added as part of the `docs-filter-list-as-string`. This is in order to handle "empty"  list as well.

---

Oh makes sense. Making the changes